assets/templates/snippets/bootstrap_form.html

```diff
@@ -28,33 +28,41 @@
             {% endif %}
         </label>
         <div class="{% if horizontal %}col-sm-10{% endif %}">
-            {# Field widget #}
-            {% if field.field.disabled %}
-                {# Readonly widget #}
-                {% render_field field class="form-control-plaintext" placeholder=field.label %}
-            {% elif form.is_bound %}
-                {# Validated widget #}
-                {% if field.errors %}
-                    {# Errors present #}
-                    {% render_field field class="form-control is-invalid" placeholder=field.label %}
+            {% if field|widget_type == "modelselect2" or field|widget_type == "modelselect2multiple" %}
```

Comment:
It's fine for now, but this is exactly what bothered me with the autocomplete2 widgets. Ideally you don't want to be conserned with the exact widget type in the template, but in order to allow bootstrap to work it's magic (and the inability to add class extras elsewhere) you kinda have to.

Reply:
This doesn't bother me. The Select2 widgets need separate CSS/HTML and the view is a good place for that. The whole file is concerned with displaying a form using Bootstrap and this fits right in since it's the same issue. I was thinking that we could do the same for the custom Bootstrap select widget